src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -235,6 +235,10 @@ private static async Task<WasmFetchResponse> CallFetch(HttpRequestMessage reques
                 JSObject fetchResponse = await BrowserHttpInterop.CancelationHelper(promise, cancellationToken, abortController, null).ConfigureAwait(true);
                 return new WasmFetchResponse(fetchResponse, abortRegistration.Value);
             }
+            catch (JSException jse)
```

Comment:
This is a breaking change, isn't it?
And we don't need to abort for JSException, right?

Reply:
This is just making the browser implementation more aligned with the desktop version, so this is fix for previous behavior difference. The test case itself is very edge case, so I don't think anybody would actually notice the improvement.

No, we don't need to abort anything. This is before the `fetch` was started.